Pheonix/Core/Source/Voxels/Blocks.cpp

```diff
@@ -28,36 +28,38 @@
 
 #include <Core/Voxels/Blocks.hpp>
 
+#include <string>
+
 using namespace pheonix::voxels;
 
-RegisteredBlock::RegisteredBlock() {};
-RegisteredBlock::RegisteredBlock(std::string Unique, int ID,
-                                 std::string Display)
-{
-	blockId     = ID;
-	uniqueName  = Unique;
-	displayName = Display;
-};
-RegisteredBlock::~RegisteredBlock() {};
+RegisteredBlock::RegisteredBlock(std::string unique, int id, std::string display) :
```

Comment:
``const std::string&`` for memory optimisation, or, since it's a constructor, use ``std::move`` in the initialiser list.